build/scripts/index.sh

```diff
@@ -20,5 +20,10 @@ for meta in "${metas[@]}"; do
     # Ignore double quotes warning for yq expression
     # shellcheck disable=SC2016,SC2094
     cat <<< "$(yq --arg metadir "${META_DIR}" '.links |= . + {self: "/\($metadir)/devfile.yaml" }' "${meta}")"  > "${meta}"
+    if [ "$(yq '.links.v2' "${meta}")" != "null" ]; then
+      cat <<< "$(yq '.links.devWorkspaces |= . +
+      {"eclipse/che-theia/latest": "/devfiles/go/devworkspace-che-theia-latest.yaml",
+      "eclipse/che-theia/next": "/devfiles/go/devworkspace-che-theia-next.next"}' "${meta}")" > "${meta}"
```

Comment:
extension should be .yaml probably

Reply:
it was commited mistakenly, reverted the fixup